manifests/m/Mozilla/Firefox/DeveloperEdition/103.0b1/Mozilla.Firefox.DeveloperEdition.installer.yaml

```diff
@@ -0,0 +1,42 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+PackageIdentifier: Mozilla.Firefox.DeveloperEdition
+PackageVersion: 103.0b1
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/devedition/releases/103.0b1/win64/en-US/Firefox%20Setup%20103.0b1.msi
+  InstallerSha256: F2C92CB54ADCE37AA63E05955265A11BF036F7D02AE1993A9B165203FE0FA1BA
+  ProductCode: '{1294A4C5-9977-480F-9497-C0EA1E630130}'
+  AppsAndFeaturesEntries:
```

Comment:
Are we sure we want to have multiple arp entries at the moment, given the issues with CLI identifying components as the base package?